include/nbl/asset/interchange/IImageAssetHandlerBase.h

```diff
@@ -178,7 +177,7 @@ class IImageAssetHandlerBase : public virtual core::IReferenceCounted
 			and texel buffer attached as well.
 		*/
 
-		static inline core::smart_refctd_ptr<ICPUImage> convertR8ToR8G8B8Image(core::smart_refctd_ptr<ICPUImage> image)
+		static inline core::smart_refctd_ptr<ICPUImage> convertR8ToR8G8B8Image(core::smart_refctd_ptr<ICPUImage> image, system::ILogger* logger)
```

Comment:
optional logger?

Reply:
Yeah i've finished the os::Printer replacement and now need to make these use optional logger